ts/Core/Axis/TreeGridTick.ts

```diff
@@ -179,6 +179,8 @@ namespace TreeGridTick {
         // Set the new position, and show or hide
         if (!shouldRender) {
             icon.attr({ y: -9999 }); // #1338
+        } else {
+            icon.attr({ y: 0 }); // #14904
```

Comment:
More compact:

`icon.attr({ y: shouldRender ? 0 : -9999 }); // #1338, #14904`

Reply:
Thank you,! I've made the change